### PR TITLE
Release v2.0.0

### PR DESCRIPTION
**Pull request by johanneshiry**
Solves #349

### CODE REVIEW

**Comment by johanneshiry** [comment]
!test

File: src/main/java/edu/ie3/datamodel/models/input/system/type/chargingpoint/ChargingPointType.java [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+  public ElectricCurrentType getElectricCurrentType() {
+    return electricCurrentType;
+  }
+
+  public Set<String> getSynonymousIds() {
+    return synonymousIds;
+  }
+
+  @Override
+  public boolean equals(Object o) {
```
[the remark this answers is not in the document]
Comment (johanneshiry) [reply src/main/java/edu/ie3/datamodel/models/input/system/type/chargingpoint/ChargingPointType.java:67]: help

[the remark this answers is not in the document]
Comment (johanneshiry) [reply src/main/java/edu/ie3/datamodel/models/input/system/type/chargingpoint/ChargingPointType.java:67]: @muse-dev help

File: src/main/java/edu/ie3/datamodel/models/input/system/EvcsInput.java [hunk 2]
```diff
@@ -9,34 +9,163 @@
...
+    return cosPhiRated;
+  }
+
+  @Override
   public EvcsInputCopyBuilder copy() {
     return new EvcsInputCopyBuilder(this);
   }
 
+  @Override
+  public boolean equals(Object o) {
```
[the remark this answers is not in the document]
Comment (johanneshiry) [reply src/main/java/edu/ie3/datamodel/models/input/system/EvcsInput.java:134]: #354

File: src/main/java/edu/ie3/datamodel/models/value/SolarIrradianceValue.java [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+  public Optional<ComparableQuantity<Irradiance>> getDiffuseIrradiance() {
+    return Optional.ofNullable(diffuseIrradiance);
+  }
+
+  public Optional<ComparableQuantity<Irradiance>> getDirectIrradiance() {
+    return Optional.ofNullable(directIrradiance);
+  }
+
+  @Override
+  public boolean equals(Object o) {
```
[the remark this answers is not in the document]
Comment (johanneshiry) [reply src/main/java/edu/ie3/datamodel/models/value/SolarIrradianceValue.java:43]: #354

**Comment by johanneshiry** [comment]
!test

**Comment by johanneshiry** [comment]
!test

**Review by ckittl — APPROVED** [review]

**Pull request merged**